torch/_inductor/codegen/mps.py

```diff
@@ -883,7 +883,11 @@ def call_kernel(self, name: str, node: Any = None) -> None:
             ]
 
             if V.graph.cpp_wrapper:
-                args.append(f"{{{', '.join(threads)}}}")
+                threads = [f"static_cast<uint64_t>({t})" for t in threads]
+                if len(threads) == 1:
+                    args.append(threads[0])
+                else:
+                    args.append(f"{{{', '.join(threads)}}}")
```

Comment:
This feels like a def/lambda one can add to codebase... (or perhaps that exists somewhere already)